Calculator/Calculator/Model/Struct/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by dragon on 2022/09/21.
+//
+
+struct CalculatorItemQueue<Item: CalculateItem> {
+    private var calculateItemList = [Item]()
+    var count: Int {
+        return calculateItemList.count
+    }
+    
+    init() {}
+    
+    mutating func enqueue(element: Item) {
+        return calculateItemList.append(element)
+    }
+    
+    mutating func dequeue() -> Item? {
+        if calculateItemList.count > 0 {
+            return calculateItemList.removeFirst()
+        } else {
+            return nil
+        }
+    }
+    
+    mutating func peek() -> Item? {
+        return calculateItemList.first
+    }
+    
+    mutating func clear() {
+        return calculateItemList.removeAll()
+    }
+    
+    func isEmpty() -> Bool {
```

Comment:
```suggestion
   var isEmpty: Bool { 
     calculateItemList.isEmpty
   }
```
요런 형태로도 쓸 수 있답니다 ㅎㅎ(파라미터를 받지 않고 있어서 변수로 처리해도 괜찮을 것 같은 케이스)

Reply:
아핳... Step2에서도 조언주셨던 것 같은데 제가 정확히 이해를 못했어서 수정은 안했던 코드같은데 이렇게 어떻게 사용하는지 알려주시니 감사합니다!!! ㅋㅋㅋ
조언주신대로 코드 수정하겠습니다~~! ㅎㅎ